src/i18n/fi.json

```diff
@@ -195,6 +196,13 @@
   "loginToComment": "Kirjaudu sisään kommentoidaksesi.",
   "loginToVoteComment": "Kirjaudu sisään äänestääksesi kommenttia.",
   "addGeojson": "Lisää geojson tiedostona",
+  "addProcess": "Lisää vaiha",
```

Comment:
Here's a small typo, should be "Lisää vaihe"

Reply:
fixed. Now all phases' titles and project names are mandatory. Other minor modifications as mentioned above are added.